cloudinary-http45/build.gradle

```diff
@@ -0,0 +1,120 @@
+plugins {
+    id 'java-library'
+    id 'signing'
+    id 'maven-publish'
+    id 'io.codearte.nexus-staging' version '0.21.1'
+    id "de.marcphilipp.nexus-publish" version "0.4.0"
+}
+
+apply from: "../java_shared.gradle"
+
+task ciTest( type: Test ) {
+    useJUnit {
+        excludeCategories 'com.cloudinary.test.TimeoutTest'
+    }
+}
+
+dependencies {
+    compile project(':cloudinary-core')
+    compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.1'
```

Comment:
don't want to bump that?

Reply:
We can bump in a separate PR, it's not really related to the client and may require changes in other modules as well.